### ISSUE TITLE
Intellisense for objects only after first character

### ISSUE THREAD

**Issue by piscopancer** [mdx-js/mdx-analyzer#434]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

v1.8.5

### Steps to reproduce

runtime: bun

### Expected behavior

using `.` for objects should use intellisense
![image](https://github.com/mdx-js/mdx-analyzer/assets/109352196/63f2cde1-17e8-4da2-a01d-0dc97dbfd087)

### Actual behavior

using `[]` works
![image](https://github.com/mdx-js/mdx-analyzer/assets/109352196/69d0db39-e287-41e0-aa68-8ba6b402ba2f)

using `.` does not work
![image](https://github.com/mdx-js/mdx-analyzer/assets/109352196/78dd112f-2552-44cd-b75f-efc2701178e1)

`.` stars working only after typing one or more characters
![image](https://github.com/mdx-js/mdx-analyzer/assets/109352196/9b83224b-cbd9-4a3f-85be-18ee900d21ab)

### Runtime

Node v21

### Package manager

Other (please specify in steps to reproduce)

### OS

Windows

### Build and bundle tools

Next.js

**Comment by remcohaszing** [comment]
Duplicate of #267
